docker/Dockerfile-ubuntu-20

```diff
@@ -42,8 +49,26 @@ WORKDIR /app/build
 
 RUN cmake .. -G Ninja -DCMAKE_BUILD_TYPE=Release -DDEV_MODE=ON && \
     cmake --build . --target all -- -j $(nproc)
+RUN make .
+WORKDIR /app
+
+COPY ./integration ./integration
+RUN cp ./build/integration/c/libotterbrix.so ./integration/csharp/src/Duckstax.Otterbrix/libotterbrix.so && \
+    cp ./build/integration/c/libotterbrix.so ./integration/csharp/tests/Duckstax.Otterbrix.Tests/libotterbrix.so && \
+    cd integration/csharp && \
+    dotnet restore ./src/Duckstax.Otterbrix/Duckstax.Otterbrix.csproj && \
+    dotnet restore ./src/Duckstax.EntityFramework.Otterbrix/Duckstax.EntityFramework.Otterbrix.csproj && \
+    dotnet restore ./test/Duckstax.EntityFramework.Otterbrix.FunctionalTests/Duckstax.EntityFramework.Otterbrix.FunctionalTests.csproj && \
+    dotnet restore ./tests/Duckstax.Otterbrix.Tests/Duckstax.Otterbrix.Tests.csproj
 
+COPY . ./
 
+WORKDIR /app/build
 RUN ctest -C -V --output-on-failure
 
-RUN cd integration/python/ && pytest
\ No newline at end of file
+RUN cd integration/python/ && pytest
+
+RUN cd .. && \
```

Comment:
@OrangeSmokingJacket @kotbegemot IMHAO I'd like to see separate docker file for dotnet. Or it should be in each docker file. 
@kotbegemot I think we need to rethink our approach to docker files infrastructure.

Reply:
So what thoughts on it?

Reply:
So what thoughts on it?